survey/__init__.py

```diff
@@ -143,7 +143,7 @@ def confirm(*args,
 
     if not 'hint' in kwargs:
         template = helpers.paint('({0}/{1}) ', _colors.hint)
-        kwargs['hint'] = utils.hint.confirm(template, default, color = color)
+        kwargs['hint'] = utils.hint.confirm(template, default)
```

Comment:
On second thought, the highlight color could be overwritten by passing a custom hint regardless. I can't come up with any scenario where the the highlight color shouldn't match the response color for implicit hints. So let's actually pass the response color here! (sorry for the roundabout 😅)